connect/src/com/telenor/connect/ConnectSdkProfile.java

```diff
@@ -0,0 +1,114 @@
+package com.telenor.connect;
+
+import android.content.Context;
+import android.net.Uri;
+import android.text.TextUtils;
+
+import com.squareup.okhttp.HttpUrl;
+import com.telenor.connect.utils.ConnectUrlHelper;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class ConnectSdkProfile extends AbstractSdkProfile {
+
+    public static final String OAUTH_PATH = "oauth";
+
+    private String clientId;
+    private String redirectUri;
+
+    public ConnectSdkProfile(
+            Context context,
+            boolean useStaging,
+            boolean confidentialClient) {
+        super(context, useStaging, confidentialClient);
+    }
+
+    @Override
+    public HttpUrl getApiUrl() {
+        HttpUrl.Builder builder = new HttpUrl.Builder();
+        builder.scheme("https");
+        builder.host(useStaging
+                ? "connect.staging.telenordigital.com"
+                : "connect.telenordigital.com");
+        return builder.build();
+    }
+
+    @Override
+    public String getClientId() {
+        return clientId;
+    }
+
+    @Override
+    public String getClientSecret() {
+        return null;
+    }
+
+    @Override
+    public String getRedirectUri() {
+        return redirectUri;
+    }
+
+    @Override
+    public String getExpectedIssuer(String actualIssuer) {
```

Comment:
I'm confused about this method. I see it's overriding and thus must have the same signature, but I don't understand why it would ever need `actualIssuer`.